docs/crafted-lisp.org

```diff
@@ -26,6 +26,27 @@ using various Lisp and Scheme variants.
   code around. ~aggressive-indent-mode~ is added to ~lisp-mode-hook~,
   ~scheme-mode-hook~ and ~clojure-mode-hook~.
 
+*** Emacs Lisp
+
+Package lint helps ensure your Emacs Lisp code follows style conventions and
+best practices. It's recommended to enable Flymake in your configuration to take
+full advantage of the Emacs linting tools:
+
+#+begin_src emacs-lisp
+(add-hook 'emacs-lisp-mode-hook #'package-lint-flymake-setup)
```

Comment:
Good question. I thought some users might find the squiggles disruptive so I figured opt-in instead of opt-out, but I use this setting myself so I'm happy to include it in `crafted-lisp-config`.